README.md

```diff
@@ -515,11 +515,11 @@ documentation](/docs/middlewares.md)
 
 ## Api
 
-## 
+## Functions
```

Comment:
This will be auto-regenerated by our build process. For some reason the new version removes the titles here, but I don't see it as a big deal for now. I will try to address this in a separate PR